packages/ida.plugin.hashdb.vm/ida.plugin.hashdb.vm.nuspec

```diff
@@ -2,11 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>ida.plugin.hashdb.vm</id>
-    <version>1.9.1</version>
+    <version>1.9.1.20240526</version>
     <authors>OALabs</authors>
     <description>Malware string hash lookup plugin for IDA Pro</description>
     <dependencies>
       <dependency id="common.vm" version="0.0.0.20240509" />
+      <dependency id="python3.vm" />
```

Comment:
I think we should use `libraries.python3.vm` instead to ensure there are no dependency issues.

Reply:
Sure thing! Updated. :)